iothub_client/inc/internal/blob.h

```diff
@@ -56,10 +56,11 @@ MU_DEFINE_ENUM_WITHOUT_INVALID(BLOB_RESULT, BLOB_RESULT_VALUES)
 * @param  httpResponse      A BUFFER_HANDLE that receives the HTTP response from the server (available only when the return value is BLOB_OK)
 * @param  certificates      A null terminated string containing CA certificates to be used
 * @param    proxyOptions    A structure that contains optional web proxy information
+* @param  curl_interface    A null terminated string containing the network interface
```

Comment:
```suggestion
* @param  curl_interface    A optional null terminated string containing the network interface
```

Reply:
Thanks for review @ericwol-msft , @jspaith 
I will take care of review comments.
Currently I am busy in some other priority work and  will raise a separate PR for c-utility shortly.
I need to understand UT framework to write unit Test-cases and i may need more time to write UT.

Reply:
done